reviews/2020-opentelemetry-annual.md

```diff
@@ -0,0 +1,108 @@
+
+Jaeger embraces OpenTelemetry with contributions and integrations. [Blog posts with details](https://medium.com/jaegertracing/jaeger-embraces-opentelemetry-collector-90a545cbc24)
+
+
+## Dev Stats
+*Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats.* 
+
+[OpenTelemetry devstats](https://opentelemetry.devstats.cncf.io/) 
+
+Notable Highlights:
+Activity per repository over 6 months shows a consistent growth https://opentelemetry.devstats.cncf.io/d/1/activity-repository-groups?orgId=1&from=now-6M&to=now
+
+There are 75 companies committing to OpenTelemetry https://opentelemetry.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20decade&var-metric=commits
+
+New PRs per week have gone from  ~80 / week  in July of 2019 to ~225-250/week in July of 2020 https://opentelemetry.devstats.cncf.io/d/15/new-prs-in-repository-groups?orgId=1&from=now-2y&to=now
+
+## Maintainers
+*How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)*
+
+There are 9 project-wide maintainers on the governance committee
+- Ben Sigelman (Lightstep)
+- Bogdan Drutu (Splunk)
+- Constance Caramanolis (Splunk)
+- Liz Fong-Jones (Honeycomb)
+- Morgan James McLean (Google)
+- Sarah Novotny (Microsoft)
+- Sergey Kanzhelev (Google)
+- Ted Young(Lightstep)
+- Yuri Shkuro (Uber)
+
+The governance committee has a rule of [maximal representation](https://github.com/open-telemetry/community/blob/master/governance-charter.md#maximal-representation) to ensure that no single company is over represented. 
+
+Each SIG has additional maintainers/approvers. Names and links to those additional names can be found [here](https://github.com/open-telemetry/community/blob/master/community-members.md)
+
+
+## Adoption
+*What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)*
+
+A list of our adopters can be found [here](https://github.com/open-telemetry/community/blob/master/ADOPTERS.md)
+
+
+## Comparison against last year
+*How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)*
+
+OpenTelemetry joined the CNCF as a sandbox project in May of last year. There is no review to compare against. Please refer to highlights section for notable accomplishments since joining CNCF. 
+
+## Goals
+*What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?*
+
+The following are “must-do” goals:
```

Comment:
Can we also include the link to the Project Roadmap? OpenTelemetry GA Launch Plan
provides a great level of details, but I am not sure if it's the only doc communicating features development to the end users/contributors.